.github/workflows/dockerimage.yml

```diff
@@ -13,23 +28,31 @@ jobs:
         uses: actions/checkout@v4
 
       - name: set up QEMU
+        if: ${{ github.ref == 'refs/heads/master' }}
         uses: docker/setup-qemu-action@v3
 
       - name: set up Docker Buildx
+        if: ${{ github.ref == 'refs/heads/master' }}
         id: buildx
         uses: docker/setup-buildx-action@v3
 
       - name: available platforms
+        if: ${{ github.ref == 'refs/heads/master' }}
         run: echo ${{ steps.buildx.outputs.platforms }}
 
-      - name: build base.app image
+      - name: build base.app image (no push)
+        if: ${{ github.ref != 'refs/heads/master' }}
+        run: |
+          docker build -t ghcr.io/umputun/baseimage/app:latest base.alpine -f base.alpine/Dockerfile
```

Comment:
the tag is confusing, and someday someone will push it and break things. I suggest dropping the tag here.